CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [x.x.x]
 
+### Added
+
```

Comment:
;)

Reply:
It has always been like this. Maybe it's not super orthodox but this way we always have the new template!

Reply:
aha alright, I see now that this is not the part on the new version. My apologies